collections/map/rpc/searchCollections.php

```diff
@@ -50,7 +50,7 @@
 
 $host = false;
 if(isset($SERVER_HOST)) {
-   $host = (str_contains($SERVER_HOST, '127.0.0.1') || str_contains($SERVER_HOST, 'localhost')? "http://": "https://") . $SERVER_HOST . $CLIENT_ROOT;
+   $host = ((str_contains($SERVER_HOST, '127.0.0.1') || str_contains($SERVER_HOST, 'localhost'))? "http://": "https://") . $SERVER_HOST . $CLIENT_ROOT;
```

Comment:
@egbot Class doesn't exist since this is off Hotfix. I can make that util change in development along side this one. The fix is in the file above this one and did this adjustment to make the code more clear but their is no functional change here